src/types.ts

```diff
@@ -6,7 +6,6 @@ export const DEFAULT_APP_NAME = 'ArDrive-Core';
 export const DEFAULT_APP_VERSION = '1.0.0';
 
 export type PublicKey = string;
-export type SeedPhrase = string;
```

Comment:
The easiest way to solve the import issue is:
 - Move this source file inside `src/types/`
 - Add this line to `src/types/index.ts`: `export * from './types.ts'`

The import tree will remain the same until we finish the types refactor when we'll surely delete/rename that file